src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java

```diff
@@ -276,7 +278,7 @@ private void activeSpeakerChanged(long ssrc)
     {
         Conference conference = getConference();
 
-        if ((conference != null) && !conference.isExpired())
+        if (conference != null)
```

Comment:
getConference() returning null equals conference has expired, but if you think it's not readable I will revert that.